freeciv-web/src/main/webapp/WEB-INF/config.properties.dist

```diff
@@ -3,6 +3,7 @@ savegame_dir=/var/lib/tomcat8/webapps/data/savegames/
 email_username=username
 email_password=password
 google-signin-client-key=122428231951-2vrvrtd9sc2v9nktemclkvc2t187jkr6.apps.googleusercontent.com
+#trackjs-token=ee5dba6fe2e048f79b422157b450947b
```

Comment:
https://github.com/freeciv/freeciv-web/blob/3059122cc0145fa966c7452d2287cd76a2bdc083/freeciv-web/src/main/webapp/WEB-INF/jsp/fragments/head.jsp#L30 will need to change too.

Reply:
Fixed. I've included the same parameter initialization as the the page that doesn't use head.jsp, and also used the same `window._trackjs`, as this is what trackjs recommends.